cmd/content-sources/main.go

```diff
@@ -3,18 +3,34 @@ package main
 import (
 	"log"
 
+	"github.com/content-services/content-sources-backend/pkg/db"
 	"github.com/gin-gonic/gin"
+	"gorm.io/gorm"
 )
 
+var DB *gorm.DB
```

Comment:
You're right, there should only be one. I didn't mean to put one in db. Where that global lives depends on how we want to call it: `DB` or `db.DB`. I don't actually use this DB global in my PR; it's just here as an example.

Reply:
i think we actually want the one in db, because the api handlers need a way to access the db connection.  We don't want the api layer importing 'main' (as the main layer is importing the api package which would create a circular dependency)

Reply:
That makes sense. I'll move it to db.